README.md

```diff
@@ -15,6 +15,8 @@ A general-purpose programming language and toolchain for maintaining
 ## Building from Source
 
 [![Build Status](https://dev.azure.com/ziglang/zig/_apis/build/status/ziglang.zig?branchName=master)](https://dev.azure.com/ziglang/zig/_build/latest?definitionId=1&branchName=master)
+[![latest release](https://img.shields.io/github/v/tag/ziglang/zig?label=latest%20release&sort=semver)](https://ziglang.org/download/)
+[![Packaging status](https://repology.org/badge/vertical-allrepos/zig.svg)](https://repology.org/project/zig/versions)
```

Comment:
Right now, the way this is formatted into the markdown is not optimal:

![image](https://user-images.githubusercontent.com/28556218/91709552-34f5f980-eb83-11ea-9cb6-b980acd61c0e.png)

Would it be possible to place this badge on the right beside the text? That way it doesn't interfere with the main content. I don't know if this is possible using pure markdown, probably some html would be needed for that

Reply:
Moved to down.